app/views/result/_people_and_organisations.html.erb

```diff
@@ -0,0 +1,6 @@
+<%
+a = side.second_head(fields)
+a.each do |person|
+%>
```

Comment:
lose the `a` variable and can this return people or organisations?

Reply:
👍